numpy/linalg/tests/test_linalg.py

```diff
@@ -876,7 +876,6 @@ class LstsqCases(LinalgSquareTestCase, LinalgNonsquareTestCase):
 
     def do(self, a, b, tags):
         if 'size-0' in tags:
-            assert_raises(LinAlgError, linalg.lstsq, a, b)
             return
```

Comment:
You can remove this early return too, I think? Or does the test below fail with that?